vagrant/provision.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -ex
+
+export DEBIAN_FRONTEND=noninteractive
+apt-get update
+apt-get --yes upgrade make awscli
+
+# Install docker
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+   $(lsb_release -cs) \
+   stable"
+apt-get install --yes docker-ce docker-ce-cli containerd.io
+usermod -aG docker vagrant
+
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+chmod 700 get_helm.sh
+./get_helm.sh
+
+PREFILIGHT_VERSION="1.9.6"
+wget "https://github.com/redhat-openshift-ecosystem/openshift-preflight/releases/download/${PREFILIGHT_VERSION}/preflight-linux-amd64"
```

Comment:
I would use make `install_preflight`. Only last 5 versions of preflight are supported for certification, so it is better to use latest in this case